pkg/arvo/app/contact-hook.hoon

```diff
@@ -138,6 +138,7 @@
     ^-  (list card)
     ::  local
     ?:  (team:title our.bol src.bol)
+      ?.  (~(has by synced) path)  ~
```

Comment:
Because the contact-hook ~(got by synced) can throw a noisy error message that doesn't matter as long as we have done the contact-view-delete.

Reply:
> we have done the contact-view-delete.

We shouldn't be calling this anymore if we've already done that, right? Or is my understanding of how we use this hook wrong?

I'm worried about this making the "we tried calling it but it's not there because of [some bug]" completely silent, harder to catch.

Reply:
If the path isn't in synced, the remove method should do nothing.